Dte.Common.Lambda/Contracts/IHandler.cs

```diff
@@ -1,9 +1,10 @@
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Dte.Common.Lambda.Contracts
 {
     public interface IHandler<in TMessage, TResponse>
     {
-        Task<TResponse> HandleAsync(TMessage source);
+        Task<TResponse> HandleAsync(TMessage source, CancellationToken? cancellationToken = default);
```

Comment:
There is no need for this to be `nullable` as it also has a default value. Remove `?`